src/editor/tests.py

```diff
@@ -242,125 +242,247 @@ def test_editor_review_round_cancel(self):
         self.assertEqual(rounds.count(), 0)
 
     def test_editor_review_review_round(self):
-        book = core_models.Book.objects.get(pk=1)
-        self.assertEqual(book.stage.current_stage == 'review', True)
+        self.assertEqual(self.book.stage.current_stage, 'review')
+
         resp = self.client.get(
-            reverse('editor_review', kwargs={'submission_id': self.book.id}))
+            reverse(
+                'editor_review',
+                kwargs={
+                    'submission_id': self.book.id
+                }
+            )
+        )
         content = resp.content
         self.assertEqual(resp.status_code, 200)
         self.assertEqual("403" in content, False)
         resp = self.client.post(
```

Comment:
Is this used at all? It looks like `resp` is reassigned a value before anything is done with it?

Reply:
No, it's not used. This happens a lot in Rua's tests.

Again, I've decided it's not worth changing this code.